docs/proposal/1-authentication-support/README.md

```diff
@@ -0,0 +1,175 @@
+kind: Secret
+metadata:
+  name: kstone-api-admin
+  namespace: kstone
+type: Opaque
+```
+
+To customize admin username and password, modify the user and password field in `kstone-api-admin` secret.
+
+#### Encryption && Decryption
+
+Use [`golang.org/x/crypto`](https://pkg.go.dev/golang.org/x/crypto) to compare a **bcrypt hashed password** with its **possible plaintext equivalent**. Returns nil on success, or an error on failure.
+
+- **bcrypt hashed password** stored in `kstone-api-admin` secret.
+- **possible plaintext equivalent** is the login password from user.
+```go
+bcrypt.CompareHashAndPassword(bcryptHashedPassword, plaintextPassword)
+```
+
+#### Customize Admin Username, Password and Private Key
+
+- Specify a new username and password(e.g. admin:admin)
+```shell
+USER=admin
+PASSWORD=admin
+```
+
+##### Customize Password
```

Comment:
If the user forgets the password, please refer to the above method to reset the password.